### PR TITLE
[SCB-378] add readme document for samples

### PR DESCRIPTION
**Pull request by lijasonvip**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: samples/auth-sample/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Sample Quick Start
+
+Auth sample use `RestTemplate` to present RSA communication between provider and consumer.
+
+* Start ServiceComb/Service Center
+
+  [how to start service center](http://servicecomb.incubator.apache.org/users/setup-environment/#)
+
+* Start auth-provider
```
Comment (WillemJiang) [inline samples/auth-sample/README.md]: Please  specify the command that we need to  use to start  the provider and consumer.

Comment (lijasonvip) [reply samples/auth-sample/README.md]: ok, will do that ASAP.

File: samples/auth-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+         address: http://127.0.0.1:30100		#service center address
+   ```
+
+2. Start the auth-provider service
+
+   - Start provider service by mvn
+
+     ```bash
+     cd incubator-servicecomb-java-chassis/
+     mvn clean install -Psamples -DskipTests			#only need to install at first time.
```
Comment (WillemJiang) [inline samples/auth-sample/README.md]: when the people see this readme, he is in the sample directory,  using the mvn clean install to compile to code. But if he wants to run main from maven , he need to know the main class name, we need to tell user for the auth-consumer at the same time.

Comment (lijasonvip) [reply samples/auth-sample/README.md]: Fixed.

**Pull request merged**